configs/captioning/coco/butd_nucleus_sampling.yml

```diff
@@ -0,0 +1,10 @@
+includes:
+- ../configs/captioning/coco/butd.yml
+model_attributes:
+  butd: &butd
+    inference:
+      type: nucleus_sampling
+      params:
+        sum_threshold: 0.8
+training_parameters:
+  batch_size: 1
```

Comment:
I'm not sure about the training aspect. For now beam search in Pythia only worked for inference. I referred to beam search config for training parameters so that nothing breaks.